src/amo/components/Addon/index.js

```diff
@@ -69,6 +69,7 @@ export class AddonBase extends React.Component {
     enable: PropTypes.func.isRequired,
     errorHandler: PropTypes.object.isRequired,
     getClientCompatibility: PropTypes.func,
+    getAddonEnabledStatus: PropTypes.func.isRequired,
```

Comment:
Thanks for renaming this function. I still don't like the name but I don't find a better name..

Reply:
ok i'll keep for now. We can update if think of something better : )

Reply:
Maybe `getEnabledState`? or we can revert back to `isAddonEnabled`...